pom.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>groupId</groupId>
```

Comment:
See [here](https://maven.apache.org/guides/mini/guide-naming-conventions.html) and [here](https://maven.apache.org/repository/guide-central-repository-upload.html) for guidelines on how to pick a groupId. There's even an example with github involved.

Reply:
Note that it makes sense to set a package for the Java classes. It should match the groupId.